baldrick/plugins/github_milestones.py

```diff
@@ -11,7 +11,9 @@ def process_milestone(pr_handler, repo_handler):
     A very simple set a failing status if the milestone is not set.
     """
 
-    mc_config = get_config_with_app_defaults(pr_handler, "milestones", {})
+    mc_config = get_config_with_app_defaults(pr_handler, "milestones", None)
```

Comment:
So I guess the question here is whether to enable the plugin one needs to just import it, or also has to have a section in the pyproject.toml file (which is what this change implies). We should make sure we are consistent across plugins in what is required for a plugin to work.

Reply:
I think this approach combined with the global config works quite well, as without it you can't configure the app to be opt in. I.e. if astroquery didn't use milestones.

Reply:
Right I see - I guess my worry is that with the new config stuff I've been working on, the app might define some defaults for all plugins. As a compromise, what if we are explicit and actually include an ``enabled`` key in the config for each plugin? That is, astropy-bot could define some defaults for if the plugin is enabled, but the plugin could still be disabled/enabled on demand.

Reply:
:+1:  I think that might be best.